src/components/welcome-page/Welcome.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../styles/variables.scss";
+
+.main-content {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    font-size: $linkedSmallFont;
+    color: $linkedDarkBlue;
+    background-color: $linkedLightBlue;
+    background: linear-gradient(0deg, rgba(40, 103, 178, 0.28895308123249297) 0%, rgba(255, 255, 255, 1) 71%);
+
+    .wrapper {
+        &__header {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            span {
+                font-size: 1.75em;
+                font-weight: 600;
+            }
+            &--icon {
+                font-size: 2em;
+            }
+        }
+        h4 {
+            padding: 0 50px;
+            font-size: 1.1em;
+            font-weight: normal;
+            letter-spacing: 1px;
+        }
+        &__btn {
+            font-size: .7em;
+            margin: 5px;
+            width: $linkedSmallFont * 7;
+            height: $linkedSmallFont * 1.5;
+            background: none;
+            border: 2px solid $linkedDarkBlue;
+            border-radius: 25px;
+            &--sign-in {
+                color: $linkedDarkBlue;
+                transition: color, background .3s;
+                &:hover{
+                    background: $linkedLightBlue;
+
+                }
+            }
+            &--sign-up {
+                background: $linkedDarkBlue;
+                transition: color, background .3s;
+                color: white;
+                &:hover{
+                    background: $linkedLightBlue;
+                }
+            }
+
+        }
+
+        &__btns {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+
+
+        }
+    }
+}
+
+@media (orientation: landscape) {
```

Comment:
We don't need to support orientation: landscape

Reply:
btw. you can nest media quesries in above code (read about nesting media queries in scss)

Reply:
In position "landscape" the buttons fit side by side and it looks better.